cabby/geo/walk.py

```diff
@@ -203,7 +205,7 @@ def get_start_poi(self,
     try:
       # Get graph that is too close (less than 200 meter path distance)
       inner_circle_graph = ox.truncate.truncate_graph_dist(
-        self.map.nx_graph, dest_osmid, max_dist=MIN_PATH_DIST, weight='length')
+        self.map.nx_graph, dest_osmid, max_dist=MIN_PATH_DIST+2*ADD_POI_DISTANCE, weight='length')
```

Comment:
nit: spacing should be `MIN_PATH_DIST + 2 * ADD_POI_DISTANCE`

Reply:
Fixed